apps/wallet-mobile/src/features/Receive/common/useMultipleAddressesInfo.tsx

```diff
@@ -0,0 +1,52 @@
+import {isBoolean, parseSafe, useAsyncStorage, useMutationWithInvalidations} from '@yoroi/common'
+import {UseMutationOptions, useQuery, UseQueryOptions} from 'react-query'
+
+import {useSelectedWallet} from '../../../SelectedWallet'
+
+const isShowingMultipleAddressInfoKey = 'isShowingMultipleAddressesModal'
+
+const useSetShowMultipleAddressessInfo = (options?: UseMutationOptions<void, Error, boolean>) => {
+  const storage = useAsyncStorage()
+  const wallet = useSelectedWallet()
+  const walletStorage = storage.join(`wallet/${wallet.id}/`)
+
+  const mutation = useMutationWithInvalidations({
+    mutationFn: (show) => walletStorage.setItem(isShowingMultipleAddressInfoKey, show),
+    invalidateQueries: [[wallet.id, isShowingMultipleAddressInfoKey]],
```

Comment:
missing wallet key, what should happen if the user switch to multiple ? should it display again the banner? cc: @SorinC6

Reply:
good question 🤔 from my understanding the `multiple addresses` modal will show only once for a user (except if the user clear the storage)

Reply:
this config is per wallet, for now is per wallet, but dunno if switch from one another what the behaviour should be, needs product confirmation.